### ISSUE TITLE
Dataloader Input / Output 설계 (__init__, __getitem__)

### ISSUE THREAD

**Issue by Ugness** [Ugness/Line-Art-Colorization-SPADE#9]
torch.utils.data.Dataset 모듈 사용할 것.
- Dataset 폴더 경로 받아서 __init__
- Normalized Lineart/Hint/Target 을 내뱉음. __getitem__
